examples/stable_diffusion_xl/gm/models/diffusion.py

```diff
@@ -102,14 +102,7 @@ def openai_input_warpper(self, cond: dict) -> Dict:
             "y": cond.get("vector", None),
         }
 
-    # TODO: Delete it
-    def _denoise(self, sigmas, noised_input, **kwargs):
-        c_skip, c_out, c_in, c_noise = self.denoiser(sigmas, noised_input.ndim)
-        model_output = self.model(ops.cast(noised_input * c_in, ms.float32), ops.cast(c_noise, ms.int32), **kwargs)
-        model_output = model_output * c_out + noised_input * c_skip
-        return model_output
-
-    def get_grad_func(self, optimizer, reducer, scaler, jit=True):
+    def get_grad_func(self, optimizer, reducer, scaler, jit=True, overflow_still_update=True):
```

Comment:
You can configure this parameter to False

Reply:
I meant that the naming conventions used in the project should be unified. Otherwise, it can quickly become very confusing if everyone uses different names for the same functionality.

Reply:
This parameter name has always been used in this project, and it may be possible to perform unified operations in sd2